xml/SkinSettings.xml

```diff
@@ -77,6 +77,12 @@
 					<onclick>Skin.ToggleSetting(no_fanart)</onclick>
 					<selected>!Skin.HasSetting(no_fanart)</selected>
 				</control>
+				<control type="radiobutton" id="606">
+					<label>$LOCALIZE[31163]</label>
+					<include>DefaultSettingButton</include>
+					<onclick>Skin.ToggleSetting(hide_background_fanart)</onclick>
+					<selected>!Skin.HasSetting(hide_background_fanart)</selected>
+				</control>
 				<control type="button" id="607">
```

Comment:
Any chance this can be merged as is? Spoke to Joe on Slack and he's not found a way to add a radio button to Custom_1105_MusicOSDSettings.xml for this, I've also had look and don't see a easy way either from my novice point of view without completely redoing Custom_1105_MusicOSDSettings.xml